src/Nethermind/Nethermind.Blockchain/Synchronization/ISyncConfig.cs

```diff
@@ -61,7 +61,7 @@ public interface ISyncConfig : IConfig
         [ConfigItem(DisabledForCli = true, HiddenFromDocs = true)]
         Keccak? PivotHashParsed => PivotHash is null ? null : new Keccak(Bytes.FromHexString(PivotHash));
 
-        [ConfigItem(Description = "Max number of attempts (seconds) to update pivot block basing on Forkchoice message from Consensus Layer. Only for PoS chains.", DefaultValue = "900")]
+        [ConfigItem(Description = "Max number of attempts (seconds) to update pivot block basing on Forkchoice message from Consensus Layer. Only for PoS chains. Infinite by default.", DefaultValue = "2147483647")]
```

Comment:
@marcindsobczak Why not some value like -1? I guess this is because setting Int.Max is quicker?

Reply:
Yeah that is what I expected to hear - we can go safe way but would be nice to leave some issue to refactor this in future

Reply:
I created low priority good first issue for refactoring it to be infinite instead of ~68 years: https://github.com/NethermindEth/nethermind/issues/5992